tools/releng/branch-repos-for-release

```diff
@@ -85,7 +86,7 @@ function branchRepo {
         # Intentionally show checkout progress for long repos, and to show
         # that a cache dir is being used.
         echo "" >&2
-        git clone git@github.com:joyent/$repoName.git "$repoCloneDir"
+        git clone git@github.com:/TritonDataCenter/$repoName.git "$repoCloneDir"
```

Comment:
This should be:

    git@github.com:TritonDataCenter/$repoName.git
                  ^
not

    git@github.com:/TritonDataCenter/$repoName.git
                  ^^

Reply:
Really?  I use git@github.com:/<org>/for ALL of my local repos.  Is the non-leading-slash the preferred/canonical form?  I mean I'm happy to change it for least-surprise reasons but is there some documentation out there?

Reply:
Without the leading slash is the canonical form. On the main repo page, if you click the green "Code" button and select ssh, it won't have the leading slash.

I honestly didn't even know it would work with a leading slash on it, but I think I'd prefer to keep it without.